### PR TITLE
KAFKA-3461: Fix typos in Kafka web documentations.

### PR DESCRIPTION
**Pull request by dongjoon-hyun**
This PR fixes 8 typos in HTML files of `docs` module. I wrote explicitly here since Github sometimes does not highlight the corrections on long lines correctly.
- docs/api.html: compatability => compatibility
- docs/connect.html: simultaneoulsy => simultaneously
- docs/implementation.html: LATIEST_TIME => LATEST_TIME, nPartions => nPartitions
- docs/migration.html: Decomission => Decommission
- docs/ops.html: stoping => stopping, ConumserGroupCommand => ConsumerGroupCommand, youre => you're

### CODE REVIEW

**Comment by dongjoon-hyun** [comment]
The Jenkins failure was `testHighConcurrencyModificationOfResourceAcls`, but it seems irrelevant to this PR. Anyway, I'll rebase this PR to trigger Jenkins test again.

```
kafka.security.auth.SimpleAclAuthorizerTest > testHighConcurrencyModificationOfResourceAcls FAILED
    java.lang.AssertionError: Should support many concurrent calls failed with exception(s) ArrayBuffer(java.util.concurrent.ExecutionException: java.lang.IllegalStateException: Failed to update ACLs for Topic:test after trying a maximum of 10 times)
        at org.junit.Assert.fail(Assert.java:88)
        at org.junit.Assert.assertTrue(Assert.java:41)
        at kafka.utils.TestUtils$.assertConcurrent(TestUtils.scala:1122)
        at 
```

**Comment by dongjoon-hyun** [comment]
At this time, the failure is the following.

```
kafka.server.OffsetCommitTest > testNonExistingTopicOffsetCommit FAILED
    java.lang.AssertionError: expected:<0> but was:<16>
        at org.junit.Assert.fail(Assert.java:88)
        at org.junit.Assert.failNotEquals(Assert.java:834)
        at org.junit.Assert.assertEquals(Assert.java:645)
        at org.junit.Assert.assertEquals(Assert.java:631)
        at kafka.server.OffsetCommitTest.testNonExistingTopicOffsetCommit(OffsetCommitTest.scala:308)
```

**Comment by dongjoon-hyun** [comment]
I rebased again to retrigger Jenkins test.

**Comment by ijuma** [comment]
Thanks for the contribution. Since you have only changed documentation, don't worry about the failures. @granthenke, looks like the new Authorizer test fails transiently. Can you please file a JIRA and take a look?

**Comment by dongjoon-hyun** [comment]
I see. Thank you, @ijuma .

**Comment by dongjoon-hyun** [comment]
Oh, finally. It passes the Jenkins.

**Comment by guozhangwang** [comment]
cc @ijuma

File: docs/connect.html [hunk 1]
```diff
@@ -297,7 +297,7 @@
...
+<code>SinkConnectors</code> usually only have to handle the addition of streams, which may translate to new entries in their outputs (e.g., a new database table). The framework manages any changes to the Kafka input, such as when the set of input topics changes because of a regex subscription. <code>SinkTasks</code>should expect new input streams, which may require creating new resources in the downstream system, such as a new table in a database. The trickiest situation to handle in these cases may be conflicts between multiple <code>SinkTasks</code>seeing a new input stream for the first time and simultaneously trying to create the new resource. <code>SinkConnectors</code>, on the other hand, will generally require no special code for handling a dynamic set of streams.
```
Comment (ijuma) [inline docs/connect.html]: Seems like there are a few cases where a space is missing after the </code> tag, eg:

``` html
<code>SinkTasks</code>should
```

**Comment by dongjoon-hyun** [comment]
Thank you, @ijuma . 
I updated the PR according to your comments.

**Comment by dongjoon-hyun** [comment]
@ijuma . Could you review this again please? :)

**Comment by ijuma** [comment]
LGTM, cc @gwenshap

**Comment by dongjoon-hyun** [comment]
Thank you, @ijuma !

**Comment by gwenshap** [comment]
LGTM. Thank you for contributing!

**Comment by dongjoon-hyun** [comment]
Thank YOU, @gwenshap ! :)

**Pull request closed without merging**